src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import propTypes, { func } from 'prop-types';
+
+const Button = ({
+  iconOnly,
+  size,
+  icon: IconComponent,
+  children,
+  ...props
+}) => (
+  <button className={'button button--' + (size ?? 'medium')} {...props}>
```

Comment:
Is `medium` as a default value necessary? Would it be better to just make the default button in size of middle and then only change the size using `small` or `big`?

See code suggestion in further comment for details